src/web/mjs/connectors/templates/WordPressMadara.mjs

```diff
@@ -89,11 +89,18 @@ export default class WordPressMadara extends Connector {
         uri.searchParams.set('style', 'list');
         let request = new Request(uri, this.requestOptions);
         let data = await this.fetchDOM(request, this.queryPages);
-        return data.map(element => this.createConnectorURI({
-            // HACK: bypass 'i0.wp.com' image CDN to ensure original images are loaded directly from host
-            url: this.getAbsolutePath(element.dataset['src'] || element['srcset'] || element, request.url).replace(/\/i\d+\.wp\.com/, ''),
-            referer: request.url
-        }));
+        return data.map(element => {
+            if (element.src.includes('data:image')) {
+                const data = element.src.split(',').pop();
+                return this._mapDataUriType(data) + data;
```

Comment:
this might as well just return element.src as queensmanga has the start bit ('data:image/jpeg;base64,')

Reply:
Maybe simplify with regex to `return element.src.match(/data:image[^\s'"]/)[0];` this should probably be enough to bypass their simple _invalid syntax but works in browser_ protection...